diem-move/diem-vm/src/parallel_executor/vm_wrapper.rs

```diff
@@ -28,8 +29,16 @@ impl<'a, S: 'a + StateView> ExecutorTask for DiemVMWrapper<'a, S> {
     type Argument = &'a S;
 
     fn init(argument: &'a S) -> Self {
+        let vm = DiemVM::new(argument);
+
+        // Load account module into the code cache.
+        #[allow(unused_must_use)]
+        {
+            vm.load_module(&ACCOUNT_MODULE, &RemoteStorage::new(argument));
```

Comment:
This is the line that did the magic! By loading `0x1::DiemAccount` into the code cache we will have a warmed up `MoveVM` by the time we hit transaction execution.

Reply:
allow(unused_must_use)] is weird, if you'd like to ignore the result, just use `let _ = ...;`